### ISSUE TITLE
installation failure due to missing perl module perl-clone

### ISSUE THREAD

**Issue by abr5988** [NBISweden/AGAT#511]
**Describe the bug**
I attempted to install AGAT in several ways, but I believe it is failing to install due to an issue with installation of the perl-clone module. 

**General (please complete the following information):**
- AGAT version v1.4.2
- AGAT installation/use: Conda
- OS: macOS
- perl version v5.32.1

**To Reproduce**
`conda config --add channels defaults 
conda config --add channels bioconda
conda config --add channels conda-forge`
install agat env first with no packages
`conda create -n agatenv
conda activate agatenv
conda install -c bioconda agat=1.4.2` getting error due to missing perl-clone;
install dependency
`conda install bioconda::perl-clone` did not work, PackagesNotFoundError

**Expected behavior**
Expected successful installation of agat

**Screenshots**
<img width="697" alt="Screenshot 2024-12-09 at 11 04 55 AM" src="https://github.com/user-attachments/assets/0c1f7275-44ae-46b4-8937-82fd7c428ad0">

**Additional context**
I also tried semi-manually installing by installing perl modules through CPAN as follows: 
`perl -MCPAN -eshell`     
install required perl modules
`install bioperl Clone Graph::Directed LWP::UserAgent Carp Sort::Naturally File::Share File::ShareDir::Install Moose YAML LWP::Protocol::https Term::ProgressBar`

However, this too failed to install the Clone module:
<img width="458" alt="Screenshot 2024-12-09 at 11 06 53 AM" src="https://github.com/user-attachments/assets/64f92fb8-f25a-48dc-bbe5-399ff5de0038">

I am a complete beginner to perl and would appreciate any help. Thank you!

**Comment by Juke34** [comment]
Yes on apple silicon chips (ARM architecture) Perl clone is not available via conda. The best choice is to use the container.
